views/guide.lavender

```diff
@@ -76,17 +76,17 @@ block content
                     'lavender/lavender': '*'
                   }
                 }
-            | then run
+            | Then, run
             code.language-markup php composer.phar install 
             | or
-            code.language-markup composer install
-            | depending on how you've installed composer. Once composer has finished 
+            code.language-markup Composer install
```

Comment:
didn't notice this, but composer doesn't need to be capitalized since it's in a code line. Code is sensitive to stuff.